packages/now-build-utils/test/unit.builds-and-routes-detector.test.ts

```diff
@@ -20,6 +20,48 @@ describe('Test `detectBuilders`', () => {
     expect(builders![0].src).not.toBe('now.json');
   });
 
+  it('should select "installCommand"', async () => {
```

Comment:
Move these tests into the second `describe()` since we're going to remove this one

Reply:
Done.